api/internal/localizer/localizer.go

```diff
@@ -76,8 +85,62 @@ func (lc *Localizer) Localize() error {
 }
 
 // TODO(annasong): implement
-// processKust returns a copy of the kustomization at kt with paths localized.
-func (lc *Localizer) processKust(kt *target.KustTarget) *types.Kustomization {
-	kust := kt.Kustomization()
-	return &kust
+// localizeFields localizes paths on the kustomization's built-in fields, excluding helm.
+func (lc *Localizer) localizeFields(_ *types.Kustomization) error {
```

Comment:
"Fields" is sort of confusing to me. My first impression is that it refer to all the string type remote URLs shown up in a kustomization.yaml file. I don't have good naming suggestions though, especially considering that the "builtin" refers to the first class transformers and this also "excluding helm". Maybe we can split this method into smaller ones when you start to implement the logic, or thinking of a more representative name.

Reply:
I've changed the name to `localizeNativeFields`.